app/views/home/index.html.erb

```diff
@@ -27,3 +27,27 @@
   </div>
 </div>
 
+<div class="container">
+  <div class="row">
+    <div class = "col-lg-12 mt-5">
+      <h1> Browse recipes </h1>
+      <ul>
+        <%= will_paginate @recipes %>
+        <% @recipes.each do |recipe| %>
+          <%= recipe.id %>
+          <div class = "row">
+            <div class="col-md-2">
+              <h5>User: <%= recipe.user.name %></h5>
+            </div>
+
+            <div class = "col-lg-6 well">
+              <h4><%= recipe.title %></h4>
+              <h5><%= recipe.ingredients %></h5>
+              <h5><%= recipe.instructions %><h5>
```

Comment:
I'm not sure the `h5` tag makes sense for either of these as they do not seem like headings. Perhaps you could use `p` tags or `div`'s with classes?

```erb
<p class="ingredients"><%= recipe.ingredients %></p>
<p class="instructions"><%= recipe.instructions %></p>
```

Reply:
Oops, yes! Good point. I actually updated this to instead follow the semantic style html `<dl><dt><dd>` instead.